gemonlinedb/include/gem/onlinedb/SystemTopology.h

```diff
@@ -0,0 +1,136 @@
+#ifndef GEM_ONLINEDB_SYSTEMTOPOLOGY_H
+#define GEM_ONLINEDB_SYSTEMTOPOLOGY_H
+
+#include <array>
+#include <vector>
+
+#include "gem/onlinedb/AMC13Configuration.h"
+#include "gem/onlinedb/AMCConfiguration.h"
+#include "gem/onlinedb/OHv3Configuration.h"
+#include "gem/onlinedb/VFAT3ChipConfiguration.h"
+
+#include "gem/onlinedb/ConfigurationTraits.h"
+#include "gem/onlinedb/PartReference.h"
+
+namespace gem {
+    namespace onlinedb {
+
+        /**
+         * @brief Describes the topology of a system.
+         *
+         * The description is built as a tree of nodes containing part
+         * references for the various components. The root node is the AMC 13
+         * (only one AMC 13 is supported), under which AMCs are found, then OHs
```

Comment:
We should support multiple AMC13s, as GE1/1 installation at p5 will have two uTCA crates

Reply:
Ok